### PR TITLE
DMN 1.3

### PR DESCRIPTION
**Pull request by philippfromme**
* generate schema from XMI
* run integration tests with DMN TCK
* update to cmof-parser@0.5.0

### Breaking Changes

* migrate to DMN 1.3
* remove biodi package

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ All notable changes to [dmn-moddle](https://github.com/bpmn-io/dmn-moddle) are d
...
 ___Note:__ Yet to be released changes appear here._
 
+* `CHORE`: generate schema from XMI ([`f4bb15`](https://github.com/bpmn-io/dmn-moddle/pull/6/commits/f4bb15627fbc434b3342c4a3db7b5ab068d7e908))
+* `CHORE`: run integration tests with DMN TCK ([`64a694`](https://github.com/bpmn-io/dmn-moddle/pull/6/commits/64a694d5e8616136af9a1ba6d147173334b3b9ac))
+* `CHORE`: update to cmof-parser@0.5.0
+
+### Breaking Changes
+
+* `FEAT`: migrate to DMN 1.3
+* `CHORE`: remove biodi package ([`2334ad`](https://github.com/bpmn-io/dmn-moddle/pull/6/commits/2334adaf3e10486e869781f30844c8def9b1b2df))
```
Comment (nikku) [inline CHANGELOG.md]: Do we tell people where / how to migrate DMN < 1.3 files? Somewhere in the readme?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,15 @@ All notable changes to [dmn-moddle](https://github.com/bpmn-io/dmn-moddle) are d
 
 ___Note:__ Yet to be released changes appear here._
 
+* `CHORE`: generate schema from XMI ([`f4bb15`](https://github.com/bpmn-io/dmn-moddle/pull/6/commits/f4bb15627fbc434b3342c4a3db7b5ab068d7e908))
+* `CHORE`: run integration tests with DMN TCK ([`64a694`](https://github.com/bpmn-io/dmn-moddle/pull/6/commits/64a694d5e8616136af9a1ba6d147173334b3b9ac))
+* `CHORE`: update to cmof-parser@0.5.0
+
+### Breaking Changes
+
+* `FEAT`: migrate to DMN 1.3
```
Comment (barmac) [inline CHANGELOG.md]: Let's add a hint regarding migration.

```suggestion
* `FEAT`: migrate to DMN 1.3. Use [`dmn-migration-utility`](https://github.com/bpmn-io/dmn-migration-utility) to convert your diagrams.
```

File: CHANGELOG.md (hunk 1 above)
Comment (nikku) [reply CHANGELOG.md]: Let's add that information at a later point in time.

File: CHANGELOG.md (hunk 2 above)
Comment (nikku) [reply CHANGELOG.md]: Added something along these lines to the CHANGELOG: https://github.com/bpmn-io/dmn-moddle/blob/master/CHANGELOG.md#breaking-changes

**Pull request merged**